### ISSUE TITLE
Setting root does not take effect

### ISSUE THREAD

**Issue by noecs** [pagekit/vue-resource#741]
I use the following code, root does not take effect.
> I noticed the description in the documentation. For root to take effect, you must use a relative path. However, I used a relative path, and it could not normally request my designated server address.
I tried to make a minimal example, it can reproduce my problem.
[https://github.com/noecs/vuetest](https://github.com/noecs/vuetest)

**Comment by noecs** [comment]
Supplementary explanation:

main.js
```javascript
import Vue from 'vue'
import App from './App.vue'

import VueResource from 'vue-resource';

Vue.use(VueResource);

Vue.config.productionTip = false;

new Vue({
  http: {
    root: 'http://google.com'
  },
  render: h => h(App),
}).$mount('#app');
```

App.vue
```javascript
<template>
  <div id="app">
    F12 for test
    <button v-on:click="onLoad">click me</button>
  </div>
</template>

<script>

export default {
  name: 'App',
  methods: {
    onLoad: function () {
      this.$http.get('test');
    }
  }
}
</script>

```

**Comment by IanT8** [comment]
Setting 
`Vue.http.options.root = 'http://google.com';`
does not seem to work either

**Comment by noecs** [comment]
I checked most of the issues. This is always a problem. This package does not seem to be maintained and updated. I will use other packages as an alternative.
@IanT8

**Comment by IanT8** [comment]
Do you intend to use axios instead?
I hit this problem doing Max Schwarzmueller's Vue.js course on Udemy.

**Comment by noecs** [comment]
Yes, I think I will encapsulate an axios that I use. Although it is not elegant and independent of vue, it should be very effective.

**Comment by noecs** [comment]
@IanT8 I didn't understand the links you provided. Unable to open the link.
